sublime-builds/Meson Compile.sublime-build

```diff
@@ -1,7 +1,9 @@
 {
 	"keyfiles": ["meson.build"],
-	
+	"name": "Meson compile"
```

Comment:
Missing comma at end of line.
The other files have the same issue.

Reply:
Checking it locally, `"name"` isn't even a valid keyword for `sublime-build` files (Sublime Text Build 4169).

Reply:
Darn, thanks! I was distracted by `variants`. I removed it now but will play with adding all to a single build system and just define the three commands as variants thereof.

Reply:
I will try to move the implementation into Python, which will make it more versatile (i.e accessing the settings file).

Reply:
Okay, cool! Then the variants approach makes even more sense because we can just access the same Python method then with different arguments!